PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -100,6 +101,12 @@ def process(self, event):
         if self.doJEC:
 #            print "\nCalibrating jets %s for lumi %d, event %d" % (self.cfg_ana.jetCol, event.lumi, event.eventId)
             self.jetReCalibrator.correctAll(allJets, rho, delta=self.shiftJEC, metShift=self.deltaMetFromJEC)
+
+        for delta, shift in [(1.0, "JECUp"), (0.0, ""), (-1.0, "JECDown")]:
```

Comment:
yes, this isn't the most optimal approach. I wanted it to be independent of doJEC. we could put the delta loop inside doJEC and use the first one as the default in this case.